blog/2024-03-21-renku-users-meeting-2024/index.mdx

```diff
@@ -11,16 +11,18 @@ image: ./renku-ecosystem.png
 
 ![Renku Users Meeting 2024](./renku_users_meeting.jpg)
 
-Do not miss the opportunity to meet other Renku Users and learn more how they leverage Renku for data science, for managing research and for teaching. Moreover, we will share Renku 2.0 status and the features prioritised for the second half of 2024.
+Do not miss the opportunity to meet other Renku Users and learn more how they leverage Renku for data science, for managing research and for teaching. Moreover, we will share Renku 2.0 status and the features prioritised for the second half of 2024. You are also invited to bring your challenges and voice your suggestions for new features in Renku.
```

Comment:
- brevity edit
- give a little context for what Renku 2.0 is
- link to our blog post about Renku 2.0

```suggestion
Do not miss the opportunity to meet other Renku Users and learn how they leverage Renku for data science, research, and teaching. Moreover, we will share updates about the upcoming new version of Renku, [Renku 2.0](https://blog.renkulab.io/renku-2/), and the features prioritised for the second half of 2024. You are also invited to voice your challenges in an open discussion about collaborative open research, and to bring your suggestions for new features in Renku.
```